test/ex_doc/formatter/epub_io_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule ExDoc.Formatter.EPUBIOTest do
+  use ExUnit.Case, async: false
+
+  @moduletag :tmp_dir
+
+  test "succeeds if trying to write into an empty existing directory", %{tmp_dir: tmp_dir} do
+    File.mkdir!("#{tmp_dir}/doc")
+
+    assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
+             generate_docs(tmp_dir)
+           end) =~ ""
```

Comment:
_any_ string would pass this assertion be we want to make sure we didn't print any warnings!

```suggestion
           end) == ""
```

Reply:
My bad here, forgot to change the operator